File: model/src/main/java/com/junius/model/vo/SysRoleQueryVo.java
//
//
package com.junius.model.vo;

import lombok.Data;

import java.io.Serializable;

/**
 * <p>
 * 角色查询实体
 * </p>
 *
 * @author qy
 * @since 2019-11-08
 */
@Data
public class SysRoleQueryVo implements Serializable {

	private String roleName;

	private  Long page;

	private Long limit;

}

File: authSystem/src/model/SysRole/SysRoleQueryVo.ts
export interface SysRoleQueryVo{
    limit:Number,
    page:Number,
    roleName:String,
}

export interface SysRoleAddVo{
    description: String,
    roleCode: String,
    roleName: String
}

export interface SysRoleUpdateVo{
    "description": String,
    "id": String,
    "roleCode": String,
    "roleName": String
}
